src/native/minipal/utils.h

```diff
@@ -38,6 +40,35 @@
 #  define FALLTHROUGH
 #endif
 
+#ifdef _MSC_VER
+#define UNREACHABLE_MSG(message) \
+    do { \
+        assert((void)message, 0); \
+        __assume(0); \
+    } while (0)
+#else
+#define UNREACHABLE_MSG(message) \
+    do { \
+        assert((void)message, 0); \
```

Comment:
```suggestion
        assert(!#message); \
        __assume(0); \
    } while (0)
#else
#define UNREACHABLE_MSG(message) \
    do { \
        assert(!#message); \
```
I do not see how you can pass two arguments to assert macro.

Reply:
I copied that from STL header. Apparently some platform with later toolset is supporting it?